src/Perpetuum.Bootstrapper/PerpetuumBootstrapper.cs

```diff
@@ -2436,14 +2453,40 @@ private void RegisterRifts()
             _builder.Register<Func<IZone, IRiftManager>>(x =>
             {
                 var ctx = x.Resolve<IComponentContext>();
-                return zone =>
+                return zone => 
                 {
                     if (zone is TrainingZone)
                         return null;
 
                     if (zone is StrongHoldZone)
                     {
-                       return ctx.Resolve<StrongholdRiftManager>(new TypedParameter(typeof(IZone), zone));
+
+                        var strongHoldExitConfigCount = Db.Query().CommandText("select count(*) from strongholdexitconfig where zoneid = @zoneId;")
```

Comment:
While we're here, style:
`SELECT COUNT(*) FROM table WHERE col=...`
Sql strings are kind of all over the place in this repo, but I prefer capital on SQL keywords
and to be consistent with below.